recipes/twitch-native-ipc/all/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.12)
```

Comment:
I see the test_package requirement was lowered, is it possible to lower this one as well?